scripts/unsupported/combine_tracks_postprocessing_cnv/combine_tracks.wdl

```diff
@@ -0,0 +1,621 @@
+# A postprocessing workflow for GATK CNV.
+#
+# THIS CANNOT BE RUN IN TUMOR-ONLY MODE.  A MATCHED NORMAL IS REQUIRED.
+#
+# Internally, we have shown that this workflow improves sensitivity and precision.  Additionally, this workflow is quite
+#  cheap to run.  Note that auxiliary files are required and without these exact files, performance may suffer.
+#
+# UNSUPPORTED and we may modify this WDL before we fully support it.
+#
+# This will also generate absolute (https://software.broadinstitute.org/cancer/cga/absolute) compatible files.
+#  Currently, the balanced-segment calling is extremely ham-fisted.
+#
+# This workflow:
+#  - Does conversion of output seg files to a format compatible with IGV.  This will be removed soon, since the GATK tools
+#     have been updated to produce files that can be easily read by IGV.
+#  - Tags possible germline events that appear in the tumor results.  This is done by comparing breakpoints and
+#     reciprocal overlaps.  This algorithm takes into account any hypersegmentation.
+#  - Tags tumor events that appear in the given blacklists (centromere and gistic blacklist)
```

Comment:
gistic -> GISTIC

Reply:
Also, can you remind me why these regions are not simply blacklisted from the outset with `-XL`?

Alternatively, we could add `-XL` to PoN creation or even use a separate `FilterIntervals` step to make things more parallel with the germline workflow.

Reply:
re: gistic ... Done

We had talked offline about keeping this wdl separate for the time being.  And that we would have a later version do the blacklisting up front.  
Basically, to provide some flexibility of blacklisting during the evaluation.  Using the `-XL` I would have to rerun the entire CNV pipeline, which would have greatly slowed the development of this pipeline.  No action.